src/h5web/visualizations/heatmap/ColorBar.tsx

```diff
@@ -1,24 +1,29 @@
 import React from 'react';
-import { scaleLinear } from 'd3-scale';
 import { AxisRight } from '@vx/axis';
 import { useMeasure } from 'react-use';
-import { adaptedNumTicks, generateCSSLinearGradient } from './utils';
+import {
+  adaptedNumTicks,
+  generateCSSLinearGradient,
+  ColorScale,
+} from './utils';
 import styles from './HeatmapVis.module.css';
 import { D3Interpolator } from './interpolators';
 
 interface Props {
-  className: string;
   interpolator: D3Interpolator;
-  dataBounds: [number, number];
+  colorScale: ColorScale;
 }
 
 function ColorBar(props: Props): JSX.Element {
-  const { className, interpolator, dataBounds } = props;
+  const { interpolator, colorScale } = props;
 
   const [gradientRef, { height: gradientHeight }] = useMeasure();
+  const scale = colorScale
+    .copy()
+    .range([gradientHeight, 0]) as typeof colorScale;
```

Comment:
`const scale: typeof ColorScale = ...` ne fonctionne pas ?

Reply:
No it doesn't... :frowning_face: 
And to be fair, the cast should not be necessary to my opinion.